pom.xml

```diff
@@ -30,6 +31,15 @@ limitations under the License.
   <packaging>hpi</packaging>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Flaky+Test+Handler+Plugin</url>
 
+  <properties>
+    <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
+    <jenkins.version>2.121.1</jenkins.version>
```

Comment:
Is there a specific reason to get a so highter version of Jenkins as baseline? the latest version 3.19 of jenkins plugin project set as baseline 2.60.1

Reply:
I think there was no specific reason. I used the latest LTS version.

Reply:
This means force all people to update jenkins. This can be an issue for docker file or server production that do not plan immediately updates so I suggest to decide the version to the parent pom